apps/examples/src/4-custom-ui/CustomUiExample.tsx

```diff
@@ -1,13 +1,15 @@
-import { Canvas, TldrawEditor, useApp } from '@tldraw/tldraw'
+import { Canvas, TldrawEditor, TldrawEditorConfig, useApp } from '@tldraw/tldraw'
 import '@tldraw/tldraw/editor.css'
 import { useEffect } from 'react'
 import { track } from 'signia-react'
 import './custom-ui.css'
 
+const config = new TldrawEditorConfig()
+
 export default function Example() {
 	return (
 		<div className="tldraw__editor">
-			<TldrawEditor autoFocus>
+			<TldrawEditor config={config} autoFocus>
```

Comment:
The `<TldrawEditor>` component needs a config prop now. You can create the configs outside of React, or else wrap the Editor in a component that provides a new config for each `<TldrawEditor>` component, as we do in the `<Tldraw>` component.